native-image-configure-examples/README.md

```diff
@@ -18,61 +18,52 @@ In both examples we use the Jackson framework to parse a JSON file to determine
 
```

Comment:
Please change the demo title to:

`# Native Image: Class Initialization at Build Time Demo`